src/features/claims/ClaimForm.tsx

```diff
@@ -73,7 +76,7 @@ export const ClaimForm = ({}: {}) => {
 
   const [selection, setSelection] = useState<{
     index: number;
-    tokenAddress: `0x${string}`;
+    tokenAddress: Hash;
```

Comment:
Assume `Hash` is just `0x${string}`. Personally prefer what is was before because this is an address not a hash.